common-graphql/src/main/scala/queries/common/ObsQueriesGQL.scala

```diff
@@ -500,6 +544,13 @@ object ObsQueriesGQL {
               }
             }
           }
+          itc {
```

Comment:
At the end are you getting ITC from the odb?

Reply:
Yes. That's where things ended up. It seems to be working pretty well.